index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <header>My logo</header>
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+    <main class="main-section">
+      <h1>
+        I'm Juan<br />
+        Glad to see you!
+      </h1>
+      <h3>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </h3>
+      <section class="second-section">
+        <h3>LET'S CONNECT</h3>
+        <ul>
+          <li>
+            <a href="#"><img src="./img/twitter.svg" alt="twitter logo" /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="./img/Linkedin icon.svg" alt="linkedin logo"
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/medium.svg" alt="medium logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/github.svg" alt="twitter logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./img/angellist.svg" alt="angel logo" /></a>
```

Comment:
- [ ] I must commend your attention to details here :clap: You have used `ul`, `li`, and `a` tags well but the `href` attributes are empty i.e. deadlinks, which is not good for user experience nor does it make your site accessible. Imagine people trying to connect with you by clicking the button without luck. Kindly include links to your respective social media for people to connect or contact you. It is your portfolio page after all! Recruiters can easily get back to you when they are impressed enough to want to engage you. By this, you won't miss out. :white_check_mark: